sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java

```diff
@@ -1583,7 +1585,8 @@ public void testUnknownTableJobGetsRemoved() throws Exception {
         try {
             execute(stmtStr);
         } catch (SQLActionException e) {
-            assertThat(e.getMessage(), containsString("Table 'doc.foobar' unknown"));
+            assertThat(e.getMessage(), containsString(String.format(Locale.ENGLISH,
+                "Table '%s.foobar' unknown", sqlExecutor.getDefaultSchema())));
```

Comment:
`schema = false` on purpose here? Asking because you have a generic call here but you don't randomize the schema.

Reply:
@mxm this is by intent because `SchemaUnknownException` would be raised before `TableUnknownException`. This test like it is now cannot be used with random schema.